tsl/src/compression/compression.c

```diff
@@ -60,6 +60,31 @@
 #include "ts_catalog/compression_chunk_size.h"
 #include "ts_catalog/hypertable_compression.h"
 
+#define DECOMPRESSION_MARKER_START "::timescaledb-decompression-start"
```

Comment:
I think if this is a marker - and someone want's to use it; it should be in the header file...but I guess this will be a secret contract between extensions; so it doesn't matter that much..
but I think it would be helpfull to have a comment somewhere to describe their usecase

Reply:
👍